### PR TITLE
Customer Case testing for database issues

### PR DESCRIPTION
**Pull request by sambible**
### Problem Statement
Test case covering 6.16 Bug surrounding database data types. This is currently in the Content View test suite as it presents itself when publishing Content Views with the repository contained, open to moving this to repository suite.

### Related Issues
https://issues.redhat.com/browse/SAT-24878

### PRT
trigger: test-robottelo
pytest: tests/foreman/api/test_contentview.py -k 'test_repository_rpm_size'

### CODE REVIEW

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_contentview.py -k 'test_repository_rpm_size'

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8864
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/api/test_contentview.py -k test_repository_rpm_size --external-logging
Test Result : ========== 1 passed, 71 deselected, 80 warnings in 2968.18s (0:49:28) ==========
```

File: tests/foreman/api/test_contentview.py [hunk 1]
```diff
@@ -1080,6 +1080,28 @@ def test_cv_audit_scenarios(self, module_product, target_sat):
         assert not self.yumcv.read().needs_publish
 
 
+@pytest.mark.tier2
+def test_repository_rpm_size(target_sat):
```
Comment (vsedmik) [inline tests/foreman/api/test_contentview.py:1084]: I would choose another test name, `test_repository_rpm_size` is a bit confusing, we don't test size of rpm repo here.
```suggestion
def test_repository_rpms_id_type(target_sat):
```

**Review by vsedmik — APPROVED** [review]
Looks good to me, one suggestion to consider.

**Review by ColeHiggins2 — APPROVED** [review]

**Pull request merged**